dev/sbtl_acc_fig_ph.py

```
@@ -0,0 +1,209 @@
+
+# Window C: supercritical, span ~ pcrit to a few pcrit (clamped to pmax).
+C_h = h_range(1.0, 3.5)
+C_p = (1.02 * p_crit, min(p_max_eos, 4.0 * p_crit))
+
+# ---------------------------------------------------------------------------
+# Backends — primed with a known-good state.
+# ---------------------------------------------------------------------------
+prime_T = max(T_triple * 1.5, 1.5 * T_min)
+prime_p = max(p_triple * 2, 1e5)
+print('Building BICUBIC...', flush=True)
+bicu = CP.AbstractState('BICUBIC&HEOS', ref); bicu.update(CP.PT_INPUTS, prime_p, prime_T)
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)